pkgs/tools/typesetting/tex/catdvi/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv
+, lib
+, fetchurl
+, fetchpatch
+, texlive
+, buildPackages
+}:
+
+let
+  buildPlatformTools = [ "pse2unic" "adobe2h" ];
+  tex = texlive.combine {
+    inherit (texlive) collection-fontsrecommended;
```

Comment:
This is not documented (yet?), but we should discourage using texlive as a build input (i.e. runtime/host dependency) unless there is a very good reason.

In this case, `collection-fontsrecommended` is a big dependency – a user may only need fewer fonts – but it is also not comprehensive, as DVIs may refer to other fonts. As a general rule, TeX Live should rather be provided by the user at runtime.

If catdvi uses `kpsewhich`, then you can simply remove `tex`, and it will work. If it uses kpathsea the C library, uhm, then maybe this should have `tlType = "bin"` and be passed to `texlive.combine` so that it gets wrapped with the desired `TEXMFCNF` – I don't believe there is another good solution. See the package `mftrace`.